packages/shared/src/components/modals/ArticlePostModal.tsx

```diff
@@ -28,15 +29,26 @@ export default function ArticlePostModal({
   ...props
 }: ArticlePostModalProps): ReactElement {
   const { showArticleOnboarding } = useContext(OnboardingContext);
-  const { post, isLoading } = usePostById({ id, isFetchingNextPage });
+  const {
+    post,
+    query: { isLoading, isFetching },
+  } = usePostById({ id });
+  const isPostLoading = isLoading || isFetchingNextPage || isFetching;
   const position = usePostNavigationPosition({
-    isLoading,
+    isLoading: isPostLoading,
     isDisplayed: props.isOpen,
     offset: showArticleOnboarding ? ONBOARDING_OFFSET : 0,
   });
+  const containerClass = 'border border-theme-divider-tertiary rounded-16';
```

Comment:
Why did we decide to place this outside the direct implementation?
This case we could even place it outside the component.

Reply:
We used it both at the `BasePostModal` to apply the same class on the container for the loading placeholder.

Reply:
But this is not exported? 👀
So it's just duplicated, not quite sure I understand why it's abstracted to a const then?

Reply:
Oh, right we can take it outside this component and be a const in the file. It is not abstracted because the article modal will have a different container class for the post page.